tests/data_files/rsa_multiple_san_uri.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Do we want to keep PEM format for consistency with other similar tests?

Reply:
I honestly don't know. On one hand we should prefer the DER format, but on the other - we're using PEM for other tests here. I'll see if changing the format to DER for all similar tests will require a larger effort.

Reply:
I changed the new tests to use .der certificates. I also noticed that the key was not uploaded in the original PR by Hannes, so I generated new ones and regenerated the certificate.

Reply:
Why the key files were added in the last commit?